apps/docs/src/guide/wallets/signing.md

```diff
@@ -1,9 +1,9 @@
 # Signing
 
-Once you've instantiated your wallet in an unlocked state using one of the previously discussed methods, you can sign a message with `wallet.sign_message`. Below is a full example of how to sign and recover a message.
+Once you've instantiated your wallet in an unlocked state using one of the previously discussed methods, you can sign a message with `wallet.signMessage`. Below is a full example of how to sign and recover a message.
```

Comment:
Out of curiosity, did we have anything like `sign_message` or `sign_transaction`?

Or was it wrong since forever?

Reply:
This has always been wrong as it was initially taken from the rust repo - #612